ocs_ci/helpers/helpers.py

```diff
@@ -3654,3 +3655,229 @@ def verify_rook_ceph_crashcollector_pods_where_rook_ceph_pods_are_running(timeou
         func=check_rook_ceph_crashcollector_pods_where_rook_ceph_pods_are_running,
     )
     return sample.wait_for_func_status(result=True)
+
+
+def induce_mon_quorum_loss():
+    """
+    Take mon quorum out by deleting /var/lib/ceph/mon directory
```

Comment:
Please handle LSO deployment case as well. The mon path for LSO is "/var/lib/rook/{mon}",

Reply:
Thanks for pointing it out, it does work here in lso based cluster ran and checked. 
But added lso_cluster validation and changed accordingly on required places.